openwisp_controller/connection/settings.py

```diff
@@ -1,6 +1,13 @@
 from django.conf import settings
 
-DEFAULT_CONNECTORS = (('openwisp_controller.connection.connectors.ssh.Ssh', 'SSH'),)
+DEFAULT_CONNECTORS = (
+    ('openwisp_controller.connection.connectors.ssh.Ssh', 'SSH'),
+    ('openwisp_controller.connection.connectors.snmp.Snmp', 'OpenWRT SNMP'),
```

Comment:
sorry, I noticed something only now, I think we should change this to be:

`openwisp_controller.connection.connectors.snmp.OpenWrtSnmp`